streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamAggregate.java

```diff
@@ -63,15 +63,14 @@ public void enableSendingOldValues() {
         private Sensor droppedRecordsSensor;
         private TimestampedTupleForwarder<K, T> tupleForwarder;
 
-        @SuppressWarnings("unchecked")
```

Comment:
As I double checked all classes that implement `KTableValueGetter`, I did some side cleanup in all files I double checked.